src/BotPlutusInterface/UtxoParser.hs

```diff
@@ -122,6 +123,10 @@ decodeHash :: Parser Text -> Parser BuiltinByteString
 decodeHash rawParser =
   rawParser >>= \parsed -> either (const mzero) (pure . toBuiltin) (tryDecode parsed)
 
+decodeHex :: Parser Text -> Parser Text
+decodeHex rawParser =
+  rawParser >>= \parsed -> either (const mzero) (pure . decodeUtf8) ((unhex . encodeUtf8) parsed)
```

Comment:
Upsetting that theres no `Hex` instance for Text, or at least none that I can find